### ISSUE TITLE
How could I  switch from one profile to another without using the mouse?

### ISSUE THREAD

**Issue by vicen1** [hultmann/multifox#175]

**Comment by ooker777** [comment]
It seems that this feature is required by many users

**Comment by hultmann** [comment]
Yes, this is a bug I want to fix when I go back to Multifox development.

**Comment by ooker777** [comment]
You are not in the development now?

**Comment by hultmann** [comment]
Nope, Multifox 3.0 is done. Let's see how it will be received, I don't know if it's worth investing more time into Multifox development.

**Comment by rahidelvi** [comment]
A keyboard shortcut to toggle between profiles would be so nice. At any rate, great Firefox extension.

**Comment by AfzalivE** [comment]
If @hultmann can give a high level overview on how to add it, I can give it a try and submit a PR
